app/client/src/widgets/ButtonWidgetV2/index.tsx

```
@@ -0,0 +1,74 @@
+import Widget from "./widget";
+import IconSVG from "./icon.svg";
+import { ButtonPlacementTypes, RecaptchaTypes } from "components/constants";
+import { BUTTON_MIN_WIDTH } from "constants/minWidthConstants";
+import { ResponsiveBehavior } from "utils/autoLayout/constants";
+import { BUTTON_COLORS, BUTTON_VARIANTS } from "@design-system/widgets";
+
+export const CONFIG = {
+  type: Widget.getWidgetType(),
+  name: "Button",
+  iconSVG: IconSVG,
+  needsMeta: false,
+  isCanvas: false,
+  searchTags: ["click", "submit"],
+  features: {
+    dynamicHeight: {
+      sectionIndex: 0,
+      active: false,
+    },
+  },
+  defaults: {
+    animateLoading: true,
+    text: "Submit",
+    buttonVariant: BUTTON_VARIANTS.FILLED,
+    buttonColor: BUTTON_COLORS.ACCENT,
+    placement: ButtonPlacementTypes.CENTER,
+    rows: 4,
+    columns: 16,
```

Noob question. why do we need `rows` and `columns` props?

---

This won't be needed in the future i guess. This is meant for a fixed layout.

---

@jsartisan Can we delete everything related to the fixed format now? Can we also enable auto layout by default and hide the toggle button to fixed one? This can be a separate task.